src/internal/connector/exchange/service_iterators.go

```diff
@@ -107,19 +97,28 @@ func FilterContainersAndFillCollections(
 			}
 		}
 
-		jobs, currDelta, err := getJobs(ctx, service, qp.ResourceOwner, cID, prevDelta)
+		jobs, newDelta, err := getJobs(ctx, service, qp.ResourceOwner, cID, prevDelta)
 		if err != nil {
-			// race conditions happen, the container might get
-			// deleted while this process in flight.
-			if errors.Is(err, errContainerDeleted) {
-				currPath = nil
-			} else {
+			// race conditions happen, containers might get deleted while
+			// this process is in flight.  If it was deleted, we skip it
+			// and move on, which will create a tombstone to delete the path.
```

Comment:
>which will create a tombstone to delete the path.

We've actually already [deleted the path from the tombstones map](https://github.com/alcionai/corso/pull/1882/files#diff-67d5d9c3856fb884fe81259eefeb5d7ddb0d695582bf44db4e9ee6e447bc808dR80) so we will not mark this as deleted

Reply:
Ack, good catch.